gammapy/time/exptest.py

```diff
@@ -0,0 +1,41 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+
+__all__ = [
+    'exptest_for_run',
+]
+
+def exptest_for_run(time_delta):
+    """Compute Mr value, the level of variability for a certain period of time.
+
+     A single Mr value can be calculated, which shows the level of variability
+     for the whole period, or the Mr value for each run can be shown.
+
+
+    Ref:Prah(1999),http://adsabs.harvard.edu/abs/1999astro.ph..9399P
+
+    Parameters
+    ----------
+    time_delta : list of float
+        the time difference between two adjacent events
+
+    Returns
+    -------
+    Mr : float
+        Level of variability
+    """
+    mean_time = np.mean(time_delta)
+    normalized_time_delta = time_delta / mean_time
+    sum_time = []
+    for i in range(len(normalized_time_delta)):
+        if normalized_time_delta[i] < 1:
+            sum_time.append(1 - normalized_time_delta[i] / 1.0)
+    mean_normalized_time=np.mean(normalized_time_delta)
+    M_value=0
+    Mr=0
+    sum_time_all = np.sum([sum_time])
+    if len(time_delta)!=0:
```

Comment:
I would suggest to remove the check `if len(time_delta)!=0` here.

Just assume callers don't pass empty data.
And if they do, your function raises a `ZeroDivisionError` ... garbage in, garbage out.

Or wait ... for this function, is returning `Mr=0` for empty data helpful?
If so, then a better way is to put:
```python
if len(time_delta) == 0:
    return 0

...
```
at the very top of the function, instead of having this special case handled further down, and the "normal" calculation indented.

OK?